msteams-platform/toolkit/deploy.md

```diff
@@ -6,120 +6,231 @@ ms.author: zhany
 
-::: zone-end
+**Sample**
 
-::: zone pivot="visual-studio"
+```text
+  - uses: cli/runDotnetCommand
+    with:
+      workingDirectory: ./src
+      execPath: /YOU_DOTNET_INSTALL_PATH
+      args: publish --configuration Release --runtime win-x86 --self-contained
+```
 
-## Deploy Teams app to the cloud using Visual Studio
+**Parameters**
 
-You can deploy the following to the cloud:
+| Parameter | description | required | default value |
+|-------------|----------|---------------|---------------|
+|workingDirectory | represents the folder where you want to run the command. If your input value is a relative path, it's relative to the workingDirectory. |No | Project root |
+|args | npm command arguments| Yes | |
+|execPath | executor path | No | System PATH |
+
+**cli/runNpxCommand**
 
-* The tab app, such as front-end apps are deployed to Azure Storage, configured for static web hosting.
-* The notification bot app with Azure Functions triggers can be deployed to Azure Functions.
-* The bot app or message extension is deployed to Azure App Services.
+**Sample**
 
-After deploying, you can preview the app in Teams client or the web browser before you start using it.
+```text
+  - uses: cli/runNpxCommand
+    with:
+      workingDirectory: ./src
+      args: gulp package-solution --ship --no-color
+```
 
-## Deploy Teams app using Teams Toolkit
+**Parameters**
+
+| Parameter | description | required | default value |
+|-------------|----------|---------------|---------------|
+|workingDirectory | represents the folder where you want to run the command. If your input value is a relative path, it's relative to the workingDirectory. |No | Project root |
+|args | command arguments| Yes | |
 
-1. Open **Visual Studio**.
-1. Select **Create a new project** or open an existing project from the list.
-1. Right-click on your project **MyTeamsApp4** > **Teams Toolkit** > **Deploy to the cloud...**.
+**azureAppService/zipDeploy**
 
-   :::image type="content" source="../assets/images/deploy-teams-app-cloud-vs/vs-deploy-cloud_1.png" alt-text="deploy to cloud":::
+**Sample**
 
-   > [!NOTE]
-   > In this scenario the project name is MyTeamsApp4.
+```text
+  - uses: azureAppService/zipDeploy
+    with:
+      workingDirectory: ./src
+      artifactFolder: .
+      ignoreFile: ./.webappignore
+      resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}}
+      dryRun: false
+      outputZipFile: ./.deployment/deployment.zip
+```
 
-1. In the pop-up window that appears, select **Deploy**.
+**Parameters**
 
-   :::image type="content" source="../assets/images/deploy-teams-app-cloud-vs/vs-deploy-confirmation.png" alt-text="Deploy to cloud confirmation dialog":::
+| Parameter | description | required | default value |
+|-------------|----------|---------------|---------------|
+|workingDirectory | Represents the folder where you want to upload the artifact. If your input value is a relative path, it's relative to the workingDirectory. |No | Project root |
```

Comment:
Sorry, I made a mistake here. It should be:
```
represents the folder where you want to upload the artifact. If your input value is a relative path, it's relative to the project root.
```

Reply:
Updated/Resolved.